demo/README.md

```diff
@@ -1,134 +1,154 @@
 ```
 
-## Running the demo with cluster mode
+## 2. Demo with The Cluster Mode
 
-```
+```bash
+# Pull the docker and start it
 docker run -it 4pdosc/openmldb:0.3.2 bash
 
 # Initilize the environment
 ./init.sh
 
-# Run feature extraction and model training. Feature extraction will read offline data from local file which may be stored in HDFS in real scene.
+# Run feature extraction and model training. Feature extraction will read offline data from the local file
 python3 train.py ./fe.sql /tmp/model.txt
 
-# Import to online database
+# Import the data to online database
 python3 import.py
 
-# Start HTTP serevice for inference with OpenMLDB
+# Start the HTTP service for inference with OpenMLDB
 ./start_predict_server.sh ./fe.sql /tmp/model.txt
 
-# Run inference with HTTP request
+# Run inference with a HTTP request
 python3 predict.py
-# the output we will see
+# The following output is expected (the numbers might be slightly different)
 ----------------ins---------------
 [[ 2.       40.774097 40.774097 40.774097 40.774097 40.774097 40.774097
   40.774097 40.774097  1.        1.      ]]
 ---------------predict trip_duration -------------
 848.014745715936 s
 ```
-To read more details about cluster mode, please refer [here](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/cluster.md)
+:bulb: To read more details about the cluster mode, please refer to the [QuickStart (Cluster Mode)](https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/cluster.md)
+
+## 3. Demo with The Standalone Mode
+**Start docker**
 
-## Running the demo with standalone mode
-### Start docker
 ```bash
 docker run -it 4pdosc/openmldb:0.3.2 bash
 ```
-### Initilize environment
+**Initialize environment**
+
 ```bash
 ./init.sh standalone
 ```
-### Create table and import the data to OpenMLDB.
+**Create table and import the data to OpenMLDB.**
+
 ```bash
+# Start the OpenMLDB CLI for the standalone mode
 ../openmldb/bin/openmldb --host 127.0.0.1 --port 6527
 ```
 ```sql
+# The below commands are executed in the CLI
 > CREATE DATABASE demo_db;
 > USE demo_db;
-> CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double,dropoff_latitude double, store_and_fwd_flag string,trip_duration int, index(ts=pickup_datetime));
+> CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double,dropoff_latitude double, store_and_fwd_flag string,trip_duration int, INDEX(ts=pickup_datetime));
 > LOAD DATA INFILE './data/taxi_tour.csv' INTO TABLE t1;
 ```
-### Run feature extraction
-```
+**Run offline feature extraction**
+
+```sql
+# The below commands are executed in the CLI
 > SET PERFORMANCE_SENSITIVE = false;
```

Comment:
`SET PERFORMANCE_SENSITIVE` is deleted in standalone quickstart.  Should keep consistent with quickstart?

Reply:
yes, i intend to delete, but we also should change the `create table`?

Reply:
yes